setup.py

```diff
@@ -45,6 +45,9 @@ def read(fname):
         'psycopg2',
         'ciso8601',
     ],
+    extras_require={
+        'faster': ['ujson', 'ciso8601']
```

Comment:
If you're making these optional deps, then they should be removed from `install_requires`.